config/app.php

```diff
@@ -157,6 +157,9 @@
         Nightwing\Providers\RouteServiceProvider::class,
 
         Nightwing\Providers\FastlyProvider::class,
+
+        Collective\Html\HtmlServiceProvider::class,
```

Comment:
Another thing that's definitely not necessary for merge, but it'd be cool to see if we can get away with not using this dependency for building forms. @weerd's got some nice ideas on how we could handle this with built-in Laravel helpers that you can take a peek at in the Gladiator app.